src/rules/requestbin.js

```diff
@@ -1,27 +1,44 @@
 /**
  * @overview Shows how to post the variables sent to your Rule to https://requestbin.fullcontact.com to help troubleshoot rule issues
  * @gallery true
- * @category enrich profile
+ * @category debugging
  *
  * Dump rule variables to RequestBin
  *
- * This rule shows how to post the variables sent to your Rule to https://requestbin.fullcontact.com to help troubleshoot issues with your Rules.
+ * > Please note: Auth0 provides [native mechanisms for debugging rules](https://auth0.com/docs/rules/current#how-to-debug-rules). Should you still desire to send internal rule variables to a third-party service, you should deactivate this rule or comment out the code once you are finished troubleshooting.
  *
- * You can run this rule by itself, or paste it into an existing rule. Once the rule has posted data to RequestBin, you can use a site like http://bodurov.com/JsonFormatter/ to more easily visualize the data.
+ * This rule shows how to post the variables sent to your Rule to https://requestbin.fullcontact.com to help troubleshoot issues with your Rules.
  *
- * > Note: You should deactivate this rule or comment out the code once you are finished troubleshooting.
+ * You can run this rule by itself, or paste it into an existing rule.
  *
- * Contributed by Robert McLaws, AdvancedREI.com
  */
 
 function (user, context, callback) {
   const request = require('request');
 
+  function filter(dict, whitelist) {
```

Comment:
why write your own "filter" instead of using `lodash.pick`?

Reply:
Absolutely no reason. Fixed.